src/render/mod.rs

```diff
@@ -182,6 +183,8 @@ pub fn render_main_window(
     )
 }
 
+// TODO Memory::allocations doesn't exist anymore
```

Comment:
Remainder to self

Reply:
oops. yea, that should be trivial to reintroduce. Do you want to add it to your rustc PR? (Again, with a comment that priroda needs this)

**alternatively** (or just do it) you can do something super evil:

Implement `AllocMap` for a new ZST type that you create. Forward every method to a `FxHashMap` that you created in a lazy static. Now you have full access to the underlying memory by accessing the lazy static ;)

Reply:
Added to my rustc PR